api/disruption_kind.go

```diff
@@ -45,6 +45,10 @@ func AppendArgs(args []string, level chaostypes.DisruptionLevel, kind chaostypes
 		args = append(args, "--on-init")
 	}
 
+	// DNS disruption configs
+	args = append(args, "--dns-server", dnsServer)
+	args = append(args, "--kube-dns", kubeDns)
```

Comment:
I think these could be gated under an `if kind == chaostypes.DisruptionKindDNSDisruption`, yes?

Reply:
Yep, I missed that, thanks.

Reply:
looks like this was added and then removed again?

Reply:
Indeed. I've pushed the change again.